rtb/piano_di_progetto_ver0.2.0.typ

```diff
@@ -9,13 +9,20 @@
   titolo: "Piano di Progetto",
   uso: "Esterno",
   versioni: (
+    "0.2.0",
+    "12/12/2025",
```

Comment:
Data sbagliata

Reply:
Sono già all'anno prossimo🤦🏻‍♂️
Comunque avevo messo il 12/12 per far sembrare fosse stata fatta l'ultima modifica inserendo la retrospettiva, essendo che per questa volta abbiamo detto che teniamo retrospettiva e planning come fossero un attività unica.
Se invece pensi sia meglio comunque mettere 2 versioni dimmelo che divido in "pianificazione" e "retrospettiva"

Reply:
giusto sarebbe più bello avere 2 versioni, la prima messa nella data di inizio sprint e la seconda in data fine sprint. Cosa ne dite? @ErPreco @DavidePicc